episodes/enabling-technologies-standards.Rmd

```diff
@@ -0,0 +1,699 @@
+                        "type": "string"
+                    }
+                },
+                
+            "testPerson": {
+                "description": "Information about the subject carrying the test device.",
+                "type": "object",
+                "required": [
+                    "height",
+                    "sex"
+                    ],
+                "properties": {
+                
+                    "height": {
+                        "description": "The height of the test person in cm (SI unit of length).",
+                        "type": "number",
+                        "minimum": 120,
+                        "exclusiveMaximum": 220
+                        },
+                        
+                    "sex": {
+                        "description": "The sex of the test person.",
+                        "type": "string",
+                        "enum": [
+                            "female",
+                            "male",
+                            "not disclosed"
+                            ]
+                        }
+                    }
+                }
+        }
+```
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: challenge
+## TASK 5: Form input & validation with JSON schema
+
+**Congratulations, you finished your metadata schema!
+Now, collecting interoperable metadata will be a lot easier in your collaboration.**
+
+We must admit: writing a valid JSON metadata record for each and every experiment that you perform is tedious and time consuming. But now that you have a JSON Schema at hand, things will get a lot easier! The project sets up a user-friendly HTML form interface for the input of JSON metadata.
+
+**Let's try this**:
+<ul>
+  <li>Download the full JSON schema here: <a href="files/exampleDataObject_schema.json" target="_blank" download>exampleDataObject_Schema.json</a></li>
+  <li>Inspect the JSON schema briefly</li>
+  <li>In your browser go to <a href="https://rjsf-team.github.io/react-jsonschema-form/" target="_blank">react-jsonschema-form playground</a>.</li> 
+  <li>Delete sample content in `JSONschema` and `formData`</li>
+  <li>Copy and paste the full schema into the `JSONschema` box</li>
+  <li>Check again if Chuck Norris properties reappeared in `formData` results; he can be tough :smile:</li>
+  <li>Inspect the form interface thoroughly.</li>
+  <li>**Optional:** Copy the final JSON object literal in `formData` in a separate text document and save the file as `exampleDataObject.json`</li> 
+</ul>  
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: hint
+## Good to know
+Note that the JSON Schema used for this demo **lacks the recommended `$schema` keyword**: this is because the playground will reject the keyword.
```

Comment:
```suggestion
Note that the JSON Schema used for this demo **lacks the recommended `$schema` keyword**: this is because the playground will unfortunately reject the keyword. You should always follow the best practices when writing a schema, but sometimes some adaptations are needed to make them work in different situations.
```